netsim/devices/cumulus_nvue.py

```diff
@@ -37,6 +37,16 @@ def nvue_check_stp_features(node: Box, topology: Box) -> None:
       more_data=err_data,
       node=node)
 
+"""
+Checks for vrf.bgp usage which is not yet implemented
+"""
+def nvue_check_vrf_bgp(node: Box) -> None:
```

Comment:
Wrong. Don't talk about "VRF BGP features" when the root cause is that you did not implement inter-VRF route leaking. You should say "netlab does not implement inter-VRF route leaking with Cumulus NVUE"

Also, intuitively it looks like using **af.vpnv4** and **af.vpnv6** is not the right way to test for that, but I won't chase those bits around with a debugger. You had a quasi-working solution for SR Linux, copying that might be better than this.

Reply:
Updated check